obspy/io/sac/sactrace.py

```diff
@@ -1030,14 +1030,17 @@ def reftime(self, new_reftime):
 
     # --------------------------- I/O METHODS ---------------------------------
     @classmethod
-    def read(cls, source, headonly=False, ascii=False, byteorder=None,
-             checksize=False, debug_strings=False):
+    def read(cls, source, encoding='ASCII', headonly=False, ascii=False,
```

Comment:
Please put the `encoding` keyword at the end of the calling signature, so that any code relying on argument positions won't fail.